recipe/meta.yaml

```diff
@@ -106,16 +101,16 @@ outputs:
         - zlib
       run:
         - zlib
-      
+
     test:
       commands:
         - command -v prismatic-gui  # [linux]
         - test -f $PREFIX/Applications/prismatic-gui.app/Contents/MacOS/prismatic-gui  # [osx]
 
   - name: pyprismatic
     build:
-      skip: True  # [win or py2k or osx]
-      string: "{{processor}}_h{{ PKG_HASH }}_{{ build }}"
+      skip: True  # [osx]
```

Comment:
I have tried different combination of selectors and none of them seem to work as expected when re-rendering. There is something I am missing here....

Reply:
Thinking a bit more about this, since the issue is that a python2.7 is added for osx, which may related to an inappropriate use of the selector - not sure the combination of the selector in the root metapackage with one of the output is supported by conda-smithy, conda-build seems to be fine with it. Python2.7 is going end of life very soon, so we could just ignore it for now and the issue should disappear soon on its own.
An alternative is to ask/report this with conda-smithy if someone have got time.

Reply:
We need a skip for py2k because otherwise the conda-forge build scripts do, occassionally, try to build for it (helpfully in a sometimes non-deterministic manner...). A belt & braces approach is useful in these scripts.